app/controllers/api/authentications_controller.rb

```diff
@@ -1,4 +1,11 @@
 module Api
   class AuthenticationsController < BaseController
+    def delete_resource(type, id, _data = {})
+      auth = resource_search(id, type, collection_class(:authentications))
+      task_id = auth.delete_in_provider_queue
```

Comment:
@abellotti yeah, I wasn't sure if we should support `DELETE`. I guess it doesn't make sense because you don't get the task information back. I will remove it from the others as well